nuget/Microsoft.CodeAnalysis.FxCopAnalyzers/LegacyRulesets/Sdl-7.1.Recommended.Error.ruleset

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0"?>
+<RuleSet Name="Microsoft SDL 7.1 Required and Recommended" Description="This rule set contains all Microsoft SDL 7.1 required and recommended rules, configured to error rather than warn." ToolsVersion="10.0">
```

Comment:
Errr...I don't remember when I committed the ruleset changes, but they're totally unrelated and should not be part of this pull request.